src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -90,7 +93,8 @@ public static Sentiment getSentiment(String projectID, String lyrics) throws Ill
     for (Entity entity : entityList) {
       // Use round() here to set the double to 2 decimals.
       SongEntity simplifiedEntity = new SongEntity(entity.getName(), 
-          Math.round(entity.getSalience() * 100.0) / 100.0, entity.getType().toString(),
+          Math.round(entity.getSalience() * 100.0) / 100.0, 
+          new HashSet<String>(Arrays.asList(entity.getType().toString())),
```

Comment:
What about changing `SongEntity` `type` field to the list of types as well, so we'll be even more flexible :D

Reply:
The `type` field for `SongEntity` is `HashSet<String>` for now. In that line of code I'm creating the default SongEntity with a single type. What do you mean changing to list of types?

Reply:
You convert your `HashSet<String>` to one `String and save it as a `type` in `SongEntity` object. We send this object to SongInfo, frontend, anywhere. So it doesn't make sense and it's not flexible, nobody will be able to operate with this types without parsing string

Reply:
So I mean that it will be good to change type of `type` field in `SongEntity` class

Reply:
Hmm take a look at the tests, type is a Set of multiple strings. Right there I'm creating a Set with only 1 string because all entities have only one type in the beginning.

Reply:
ok, now I see it, got it, thanks)